vm/devices/net/mana_driver/src/gdma_driver.rs

```diff
@@ -199,14 +199,11 @@ impl<T: DeviceBacking> Drop for GdmaDriver<T> {
         }
 
         // Wait for VF ownership of the shared memory
-        const BEFORE_DESTROY_HWC: &str = "Before GdmaDriver destroy HWC";
-        let _ = match self.wait_for_vf_to_own_shmem().context(BEFORE_DESTROY_HWC) {
-            Ok(hdr) => hdr,
-            Err(e) => {
-                tracing::error!(context = BEFORE_DESTROY_HWC, error = %e);
-                return;
-            }
-        };
+        let header = self.wait_for_vf_to_own_shmem().context("Before GdmaDriver destroy HWC");
+        if let Err(e) = header {
+            tracing::error!(context = %e.to_string(), error = ?e.root_cause());
```

Comment:
Do you need to log context and error separately? Dont you get the same outcome if you just log `tracing::error!(e, " "Wait for VF posession to post HWC destroy")`? In this case, you also dont need a `.context()` as you are logging here directly and not rolling up the errors.